src/api.js

```diff
@@ -20,3 +20,15 @@ export const getCommentsByArticleId = (articleId) =>{
         return res.data.comments
     })
 }
+
+export const patchArticleVotes = (articleId, amount) =>{
+    if(navigator.onLine){
+    return newsApi.patch(`/articles/${articleId}`, {inc_votes:amount}).then((res)=>{
+        })
+    .catch((err)=>{
```

Comment:
do you need to catch here? You should catch at the end of your chain. You've caught, just to throw again confusingly.